presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,68 @@
+* Fix typo in high memory task killer strategy config `experiemental.task.high-memory-task-killer-strategy` to `experimental.task.high-memory-task-killer-stragtegy`.
+* Improve history optimizer latency for queries which timeout during query registration in history optimizer.
+* Improve latency of history based optimizer.
+* Add High Memory Task Killer which troggers if worker is running out of memory and garbage collection not able to reclaim enough memory. The feature is enabled by `experimental.task.high-memory-task-killer-enabled`.  There are two strategies for the task killer which can be set via `experiemental.task.high-memory-task-killer-strategy`. (default: `FREE_MEMORY_ON_FULL_GC`, `FREE_MEMORY_ON_FREQUENT_FULL_GC`).
+* Add a session property `history_input_table_statistics_matching_threshold` to set the threshold for input statistics match in HBO.
+* Add an optimization, which removes the redundant cast to varchar expression in join condition. The optimizer is controlled by session property `remove_redundant_cast_to_varchar_in_join` and enabled by default.
+* Add config to enable memory based split processing slow down. This can be enabled via `task.memory-based-slowdown-threshold`.
+* Add optimization for scaled writers with HBO, it can improve latency for query with scaled writer enabled. It's controlled by session property `enable_hbo_for_scaled_writer` and default to false.
+* Added links for the REST API resource pages to the Presto documentation.
+* Added type widening support to the parquet column readers (int, bigint, and float) to be able to perform type conversions between inmediate numerical types.
+* Change PushPartialAggregationThroughExchange optimization to use partial aggregation statistics when available, and when use_partial_aggregation_history=true.
+* Enable optimization of values node followed by false filter to empty values node.
+* Enhance join-reordering to work with non-simple equi-join predicates.
+* Extends HBO to track statistics of execution partial aggregation nodes.
+* Free memory on frequent full garbage collection strategy triggers task killer if worker is having frequent garbage collection and enough memory is not reclaimed during the garbage collections. Frequent garbage collection is identified via `experimental.task.high-memory-task-killer-frequent-full-gc-duration-threshold` and reclaim memory threshold is configured via `experimental.task.high-memory-task-killer-reclaim-memory-threshold`.
+* Free memory on full garbage collection strategy triggers task killer if worker is running low memory and full GC is not able to reclaim enough memory. Low memory threhold is set by  `experimental.task.high-memory-task-killer-heap-memory-threshold`  and Full GC reclaim memory threhold is set by `experimental.task.high-memory-task-killer-reclaim-memory-threshold`.
+* Iceberg connector now supports ANALYZE when configured with a Hive catalog and the table is unpartitioned.
```

Comment:
We should move Iceberg updates under the `Iceberg Changes/Connector` below.

Reply:
moved